resources/views/members/map.blade.php

```diff
@@ -7,56 +7,69 @@
 
 @section('content')
 
-<h1>Ledenkaart</h1>
+	<h1>Ledenkaart</h1>
 
-<hr/>
+	<hr/>
 
-<p><b>Noot</b>: de markers worden een voor een geladen omdat Google een limiet stelt aan het aantal queries per seconde. Je moet dus even geduld hebben.</p>
+	<p>Legenda: rood = normaal, groen = aspirant, blauw = info.</p>
 
-<p>Legenda: rood = normaal, groen = aspirant, blauw = info.</p>
-
-<div id="map-canvas" style="width:100%; height:1000px; margin-bottom:10px;"></div>
+	<div id="mapdiv" style="width:100%; height:750px; margin-bottom:10px;"></div>
 
 @endsection
 
+@section('script')
+
+	<script type="text/javascript">
+
+	function getColoredIcon(color) {
+		const markerHtmlStyles = `
+			background-color: ${color};
+			width: 3rem;
+			height: 3rem;
+			display: block;
+			left: -1.5rem;
+			top: -1.5rem;
+			position: relative;
+			border-radius: 3rem 3rem 0;
+			transform: rotate(45deg);
+			border: 1px solid #FFFFFF`
+
+		return L.divIcon({
+			// className: "my-custom-pin",
+			iconAnchor: [0, 24],
+			labelAnchor: [-6, 0],
+			popupAnchor: [0, -36],
+			html: `<span style="${markerHtmlStyles}" />`
+		});
+	}
+
+	const redIcon = getColoredIcon('red');
+	const greenIcon = getColoredIcon('green');
+	const blueIcon = getColoredIcon('blue');
 
-@section('header')
-<!-- Load Google maps script -->
-<script type="text/javascript" src="https://maps.googleapis.com/maps/api/js"></script>
+	// Setup map
+	var map = L.map('mapdiv').setView([51.505, 5.4], 7);
 
-<script type="text/javascript">
-function initialize() {
-	// First initialize map
-	var geocoder = new google.maps.Geocoder();
-	var mapOptions = {
-		center: { lat: 52.31, lng: 5.55},
-		zoom: 8
+	L.tileLayer('https://{s}.tile.openstreetmap.org/{z}/{x}/{y}.png', {
+		attribution: '&copy; <a href="https://www.openstreetmap.org/copyright">OpenStreetMap</a> contributors'
+	}).addTo(map);
+
+	const markers = {!! json_encode($markers) !!}
```

Comment:
You can use the  `@json ` blade directive. Which is less vulnerable to nasty injection issues.